infrastructure/main.tf

```
@@ -149,6 +154,8 @@ module "citizen-frontend" {
 
     CONTACT_EMAIL = "${data.azurerm_key_vault_secret.staff_email.value}"
 
+    //LaunchDarkly
```

either 
```suggestion
    // LaunchDarkly
```
or remove this line, not sure if comment is needed